src/AEPi/codecs/Tex2ImgCodec.py

```diff
@@ -1,6 +1,7 @@
 from typing import Optional
 
 from PIL import Image
+import etcpak
```

Comment:
Please could we call etcpak from EtcPakCodec instead? We can just leave Tex2ImgCodec as only supporting `decompress=[CompressionFormat.ATC]`, and get rid of `TEX2IMG_FORMAT_MAP` altogether :)